getting-started/login-for-multi-account-users.md

```diff
@@ -0,0 +1,29 @@
+# Login for multi-account users
+
+You can have multiple Imagekit accounts associated with the same email. For a multi-account user, the login flow will be as follows:
```

Comment:
You cannot have multiple ImageKit accounts with the same email. You can have multiple users in different account with the same email.

Reply:
A user can be associated with multiple ImageKit accounts using the same email address. A previously registered user can be added to a different account. Read how to add a user to an account [here](./user-access-management/).

The login flow for a multi-account user is as follows:

Reply:
done